tests/bench_timers/Makefile

```diff
@@ -0,0 +1,54 @@
+include ../Makefile.tests_common
+
+# These boards only have a single timer in their periph_conf.h, needs special
+# CFLAGS configuration to build properly
+SINGLE_TIMER_BOARDS = \
+  native \
+  nucleo32-f031 \
```

Comment:
This PR still uses the old nucleo board names. Here it should be nucleo-f031k8 and below nucleo-f042k8